connector/connect/common/src/main/protobuf/spark/connect/commands.proto

```diff
@@ -298,6 +306,15 @@ message StreamingQueryCommandResult {
     // Logical and physical plans as string
     string result = 1;
   }
+
+  message ExceptionResult {
+    // Exception message as string
+    optional string exception_message = 1;
+  }
+
+  message AwaitTerminationResult {
+    optional bool terminated = 1;
```

Comment:
QQ; why is it `optional`? I thought it's always true or false?

Reply:
+1 it is wired to see the only field which is a bool field is optional. It should be either terminated or not terminated.

Reply:
Ah before I put it to be optional because I wanted it return nothing in `awaitTermination()` because it returns nothing. But since we made it return a true even in `awaitTermination()` I could remove the optional tag

Reply:
Lets remove optional. Better to always set it.